stock_pack_operation_quick_lot/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Pack Operation Quick Lot',
+    'summary': 'This module allows set pack operation Lot Name and '
```

Comment:
Short this with `Set lot name and end date directly on picking operations`

Reply:
[add] This module is for creating lot on incoming receptions